bokeh/io/export.py

```diff
@@ -269,8 +269,11 @@ def is_bokeh_render_complete(driver):
 
 @dev((1,0,0))
 def terminate_web_driver(driver):
-    driver.service.process.send_signal(signal.SIGTERM)
-    driver.quit()
+    try:
+        driver.service.process.send_signal(signal.SIGTERM)
+        driver.quit()
```

Comment:
At this point I'm not sure why we do both, but it makes sense to ignore errors, when web driver was terminated semi-gracefully.